examples/c/example2.c

```diff
@@ -217,7 +217,7 @@ init_logging(int my_rank, int event_num[][NUM_EVENTS])
  *
  * @return 0 if example file is correct, non-zero otherwise. */
 int check_file(int ntasks, char *filename) {
-    
+#ifdef _NETCDF
```

Comment:
#ifdef _NETCDF is never necessary. NetCDF is required for the build. PIO no longer builds with pnetcdf only.